third_party/breakpad.cmake

```diff
@@ -0,0 +1,195 @@
+# Copyright (C) 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set(breakpad_dir "third_party/breakpad/src")
+
+if(APPLE)
+    set(breakpad_srcs
+        "common/convert_UTF.c"
+        "common/md5.cc"
+        "common/simple_string_dictionary.cc"
+        "common/string_conversion.cc"
+        "client/minidump_file_writer.cc"
```

Comment:
Try to keep these alphabetically sorted.

Reply:
done.